### PR TITLE
Improve dominant frequency change example

### LINKED ISSUE #655: [DOCS] Improve Dominant Frequency Change Check notebook
Follow the guidelines on #543

### PR DESCRIPTION
**Pull request by ItayGabbay**
#### Reference Issues/PRs
Closes #655 

#### Any other comments?

---
Thanks for contributing a pull request! Please ensure you have taken a look at
the contribution guidelines: https://github.com/deepchecks/deepchecks/blob/main/CONTRIBUTING.rst

### CODE REVIEW

File: docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py [hunk 1]
```diff
@@ -2,43 +2,58 @@
...
+* `Generate Data <#generate-data>`__
+* `Run The Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
 
-#%%
-# Generating Data
-# ===============
+What is a Dominant Frequency Change?
+====================================
+Dominant Frequency Change is a data integrity test which simply checks whether dominant values have increased
```
Comment (nirhutnik) [inline docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:16]:
```suggestion
Dominant Frequency Change is a data integrity check which simply checks whether dominant values have increased
```
come on gabbay lol

File: docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py [hunk 2]
```diff
@@ -2,43 +2,58 @@
...
+significantly between the test data and a reference data.
```
Comment (nirhutnik) [inline docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:17]:
```suggestion
significantly between test data and train data.
```
I agree it doesn't have to be train, but we do call it train, so we should be consistent

File: docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py [hunk 3]
```diff
@@ -2,43 +2,58 @@
...
 
-iris = load_iris(return_X_y=False, as_frame=True)
-X = iris.data
-y = iris.target
-X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=55)
-train_dataset = Dataset(pd.concat([X_train, y_train], axis=1), 
-            features=iris.feature_names,
-            label='target')
+This check compares the dominant values of each feature in the test data to the dominant values of the same feature in
+the reference data. If the ratio of the dominant values of the test data to the reference data is greater than a
```
Comment (nirhutnik) [inline docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:20]:
```suggestion
the reference data. If the ratio of the test to train dominant values is greater than a
```

File: docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py [hunk 4]
```diff
@@ -2,43 +2,58 @@
...
+configurable threshold, the check fails. This threshold can be configured by specifying the `ratio_change_thres`
```
Comment (nirhutnik) [inline docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:21]:
```suggestion
threshold, the check fails. This threshold can be configured by specifying the `ratio_change_thres`
```
If you say that it's configurable in the next sentence, no need to say it in this one

**Review by nirhutnik — APPROVED** [review]

File: docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py [hunk 5]
```diff
@@ -2,43 +2,58 @@
...
+from deepchecks.tabular.datasets.classification import iris
+#%%
+# Generate data
+# =============
+train_ds, test_ds = iris.load_data(data_format='Dataset', as_train_test=True)
 
 #%%
-# Running ``dominant_frequency_change`` check
-# ===========================================
+# Introducing duplicates in the test data
```
Comment (shir22) [inline docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:39]:
```suggestion
# Introducing Duplicates in the Test Data
```

File: docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py (hunk 3 above)
Comment (shir22) [inline docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:20]:
```suggestion
the train data. If the ratio of the test to train dominant values is greater than a
```

Comment (shir22) [reply docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:20]: (extended Nir's fix also to include the additional "reference")

File: docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py [hunk 6]
```diff
@@ -2,43 +2,58 @@
...
+* `Run The Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
 
-#%%
-# Generating Data
-# ===============
+What is a Dominant Frequency Change?
+====================================
+Dominant Frequency Change is a data integrity test which simply checks whether dominant values have increased
+significantly between the test data and a reference data.
```
Comment (shir22) [inline docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py:18]: Missing here a sentence about "motivation" - what does this check try to catch (not in a technical manner but in a conceptual manner - a sharp increase in a common null/constant value etc...)

**Review by shir22 — COMMENTED** [review]
Can you please also append the html so we can see it?
(in general in major docs updates...)

**Comment by ItayGabbay** [comment]
[Dominant Frequency Change — Deepchecks 0.5.0.dev0-219-g4fd87f97 documentation.pdf](https://github.com/deepchecks/deepchecks/files/8527564/Dominant.Frequency.Change.Deepchecks.0.5.0.dev0-219-g4fd87f97.documentation.pdf)

**Pull request merged**